tendrl/node_agent/ceph_integration/flows/import_cluster.py

```diff
@@ -27,8 +30,14 @@ def run(self):
                                            json.dumps(job))
         if self.node_id in node_list:
             self.parameters['fqdn'] = socket.getfqdn()
-            ceph = "git+https://github.com/Tendrl/ceph_integration.git@v1.0"
-            self.parameters['Package.name'] = ceph
+            if config.get("node_agent", "deployment_type") == "production":
```

Comment:
All of this hardcoding is supposed to be in the definition YAML files, and not in the code.